apps/remix-ide-e2e/src/helpers/init.ts

```diff
@@ -27,30 +27,36 @@ export default function (browser: NightwatchBrowser, callback: VoidFunction, url
     })
     .verifyLoad()
     .perform(() => {
+
       if (hideToolTips) {
+        
         browser.execute(function () { // hide tooltips
           function addStyle(styleString) {
+            
             const style = document.createElement('style');
             style.textContent = styleString;
             document.head.append(style);
           }
 
           addStyle(`
-            .bs-popover-right {
-              display:none !important;
-            }
-            .bs-popover-top {
-              display:none !important;
-            }
-            .bs-popover-left {
-              display:none !important;
-            }
-            .bs-popover-bottom {
+            .popover {
               display:none !important;
             }
           `);
         })
-      }
+    }})
+    .perform(() => {
+      browser.execute(function () { 
+        (window as any).logs = []
+        console.log = function () {
```

Comment:
is this a new feature for the CI?

Reply:
yeah you can fetch the logs then in any test. helpful but maybe not necessary for this PR.